loader/src/main/kotlin/net/weavemc/loader/bootstrap/transformer/SafeTransformer.kt

```diff
@@ -24,18 +29,23 @@ internal interface SafeTransformer : ClassFileTransformer {
         classfileBuffer: ByteArray
     ) = runCatching {
         val bytes = transform(loader, className, classfileBuffer)
+        if (dumpBytecode && bytes != null) {
+            klog.trace("Dumping transformed bytecode for {}", className)
+            Path(".weave-transformed/$className.class").also {
+                it.parent.createDirectories()
+            }.writeBytes(bytes)
+        }
```

Comment:
Erm `InjectionHandler`?

Reply:
I definitely also missed that, but doesn't it only apply to hook transformations? Should I keep it in SafeTransformer and remove the one in InjectionHandler since it's broader?

Reply:
We could, but that is super irrelevant for a logger pr lol

Reply:
true lmao